### PR TITLE
moved to old folder

### PR DESCRIPTION
**Pull request by dev-sankhadip**
#4 
Project moved to old folder

### CODE REVIEW

**Comment by thebishaldeb** [comment]
Even the .gitignore file needs to be inside.

**Comment by thebishaldeb** [comment]
Update the Readme as directed in the issue.

**Comment by dev-sankhadip** [comment]
Please close this pull request, so that i can create another PR

**Comment by thebishaldeb** [comment]
You can update here itself. Just push the commit. Don't need to create another PR. Also make sure the old site runs.

**Comment by dev-sankhadip** [comment]
I have updated and pushed commit, check it now. Is it ok?

**Comment by thebishaldeb** [comment]
I did not ask to do that. I asked to update readme as is there in the issue #4

**Comment by thebishaldeb** [comment]
Please read the issue carefully and do the changes in readme. I did not ask to give the tutorial in the readme.

**Comment by thebishaldeb** [comment]
Update the line
```
cd TheZalophusBlog
```
to 
```
cd TheZalophusBlog/old
```
I did not ask to add those lines.

**Comment by dev-sankhadip** [comment]
Sorry bro i am messing it up.

**Comment by thebishaldeb** [comment]
It is fine. You just need to change that line, nothing else. Do it, I'll merge it.

**Comment by dev-sankhadip** [comment]
Is it ok now? Or still need to be changed?

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@
 * Create pull requests, submit bugs, suggest new features.
 <br>
 
+## Todo
+* Create a new folder called Old in the repo and move everything except README and LICENSE in that folder.
+
```
Comment (thebishaldeb) [inline README.md]: Not needed.

Line 38 Update
```
cd TheZalophusBlog
```
to 
```
cd TheZalophusBlog/old
```

**Comment by dev-sankhadip** [comment]
I updated it in line 38

**Comment by thebishaldeb** [comment]
You could hvae directly updated that line itself, rather than adding a new section. It is still fine though. Would do it later.

**Comment by dev-sankhadip** [comment]
I think this time i caught it right....

**Comment by thebishaldeb** [comment]
Not really. But yeah it is okay.

**Comment by thebishaldeb** [comment]
Our focus should be on the other works.

**Comment by dev-sankhadip** [comment]
I am gonna work on that. I irritated you lot for this simple thing. :)

**Comment by dev-sankhadip** [comment]
Am i gonna get points for this?

**Comment by thebishaldeb** [comment]
I added the points already. And it's fine, you don't have to be sorry.

**Pull request merged**